### ISSUE TITLE
Accessibility Review of COE experience from VA Benefits & Claims

### ISSUE THREAD

**Issue by Midge-dev** [department-of-veterans-affairs/va.gov-team#60647]
## About your team

- Team name: 
-	OCTO-DE product owner: Matt Self matthew.self2@va.gov
-	Product manager: Laura Steele laura.steele@coforma.io
-	Designer: 
-	FE engineer: Sean Midgley sean.midgley@oddball.io
- Accessibility specialist:
- Content writer: 
-	Product/team Slack channel: #benefits-ce-non_disability_exp
 
### What does your team need support for? Check all that apply.
- [ ] Launching one or more unauthenticated pages, online forms, apps, or tools on VA.gov
- [ ] Making revisions to one or more existing VA.gov pages
- [ ] Accessibility review of wireframes and/or prototypes
- [x] Accessibility issue (general request)
- [ ] Accessibility PDF review request
- [ ] Accessibility user research support ([submit a separate, additional request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name))
- [x] Something else 
 
### Give a brief description of what your team needs support with:

My team needs support with ...
A new accessibility review based on [this issue](https://github.com/department-of-veterans-affairs/va.gov-team/issues/23562) from a couple years ago. We want to confirm wether or not an accessibility issue still exists, and if so, we want understand how we should go about fixing them (VA best practices, naming conventions, etc.)
 
### Will this new product be released incrementally (for instance 25% of users initially)? 
  - [ ] Yes
  - [ ] No
 
**Note:** If you checked yes, we'll reach out to discuss details about the content in the react widget.
  
### When do you expect to launch your product to 100% of users?
  
*Please provide an estimated date so our team can create other relevant tickets.*
 
### Will this work be going through the Collaboration Cycle?
- [ ] Yes 
- [ ] No
 
#### If yes, please check all of the phases in the Collaboration Cycle you have completed: 
- [ ] Design Intent
- [ ] Midpoint Review
- [ ] Staging
 
### When does this work need to be completed?
- Estimated launch date: TBD
- Estimated staging review date: TBD
- Content, Accessibility, and IA work needed by: Earliest convenience
  
## Supporting artifacts
*Please provide supporting artifacts as available.*

- Link to product outline:
- Link to or attach user flows:
- Link to prototype or mockups:
- Link to your research plan: 
- Link to any epics/issues that may be helpful: [Accessibility Feedback for Design Intent Review: LGY: Certificate of Eligibility #23562](https://github.com/department-of-veterans-affairs/va.gov-team/issues/23562)

#### Do you plan to bring this to an upcoming content office hours session?
- [ ] Yes, benefit content office hours (Thursdays, 3:00 p.m. to 3:45 p.m. ET)
- [ ] Yes, health  content office hours (Thursdays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] Yes, unauth office hours (Mondays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] No, but I'd like to schedule time to talk about this request
- [x] No, let's work asynchronously and meet if needed

 **Note:** If we think this work would benefit from a collaborative session with you, we may ask you to bring it to office hours or set up a separate time to meet.
 
### Accessibility Help in Slack
The [#accessibility-help channel](https://dsva.slack.com/archives/C8E985R32) in the DSVA Slack, is also available to you as a resource. Everyone is welcome to ask questions or get help from our a11y (accessibility) specialists. Tag `@Terry Nichols` to get a11y help asap.

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#sitewide-content-ia](https://dsva.slack.com/channels/sitewide-content-ia) Slack channel and tag Randi Hecht.**

- If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
- If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)
 
 - If you are creating an [experimental design,](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) please [also contact the DST. ](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- If you are requesting [a11y support for user research,](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name) please complete this form, next.

**Comment by sara-amanda** [comment]
## CAIA Update

### Replicating the Issue
:warning: **Is this issue currently occurring in production?** 
- [ ] **Yes:** 
- [ ] **No**

_If yes, can you provide the location of this issue, so that we can check and see if we can replicate the issue?_   

### Larger Issue
**This appears to be part of a larger series of list loop issues^. Below are some that I was able to locate.** 
- **va-forms-system-core issue:** [(Needs review and refinement) List loop pattern #38](https://github.com/department-of-veterans-affairs/va-forms-system-core/issues/38)
- **vets-design-system-documentation:** [Experimental Design List & Loop #1187](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1187)
- **va.gov-team:** [508-defect-2 [COGNITION, SCREEN READER]: List loop buttons and fields SHOULD have descriptive labels #19696](https://github.com/department-of-veterans-affairs/va.gov-team/issues/19696)

### Past Recommendations
Have any and/or all of the following "**[should](https://github.com/department-of-veterans-affairs/va.gov-team/issues/23562#issue-863932950)**" recommendations from @joshkimux been completed? 

Please check the box, if **yes**, and leave empty, if **no**.
 
#### **Question Rationale**
- [ ] For the question "have you received a purple heart", it was strongly recommend a brief note be provided, or reason explaining why the VA is asking for this or why they don't already know this.
#### **Purpose & Link Destination**
- [ ] It was recommended that the following be reworded: `Please go to your profile page` link to include the purpose of said link. For example, `Go to your profile page to update your contact information` provides both a purpose and a destination.
#### **Description List Needed**
- [ ]  Step 3 of 4: Your VA loan history should have a Veteran's loans [written as a description list](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl).
#### **One-question per page approach** 
- [ ] (Reference item:[ structuring forms gov.uk process](https://www.gov.uk/service-manual/design/form-structure#start-with-one-thing-per-page)) Split up Step 1 of 4: Your VA loan history into multiple steps since it currently has too many questions including: date fields, address fields, and yes/no fields.
#### **For Developers**
- [ ] Add aria-hidden="true" to the "!" icon if it is kept in the final version. The bolded "note:" provides a text equivalent, so the "!" should be hidden to screen reader users.

### Reference Item^
#### Patterns that we call a "list-loop" per @Mottie 

- `Array field (single page list loop)` - deprecate pattern since it's not screen-reader friendly
- `Array "add item" list loop` - multiple pages, but the "summary" page is at the beginning, clicking add or edit will redirect the Veteran to a page that then returns them to the summary page.
- `Array data list loop` - multiple pages, but does not return to any previous pages - a summary page is usually added at the end of the pattern; this will be the likely candidate we'll use for an "internal" sub-task pattern, but on the summary page, we'll include links to return to a specific data page, but the Veteran will need to step through each page to get back to the summary

cc: @coforma-terry @SarahKay8

**Comment by sara-amanda** [comment]
## CAIA Update

> RE: My team needs support with ...
> A new accessibility review based on https://github.com/department-of-veterans-affairs/va.gov-team/issues/23562 from a couple years ago. We want to confirm wether or not an accessibility issue still exists, and if so, we want understand how we should go about fixing them (VA best practices, naming conventions, etc.)

### Experimental Design

@Midge-dev in addition to the items @joshkimux recommended in my comments above, you can also take a look at this experimental design Pam did (linked above and below). I am adding more information relating to this, below and links.
- [Experimental Design List & Loop #1187](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1187) 
    - I am told that it is not built into the form system, so it will be a little bit of work to reproduce this. 
        - However, @briandeconinck recommends checking this is in production now (it should be). 
    - **Point of Reference**: [The Supplemental Claim form in staging](https://staging.va.gov/decision-reviews/supplemental-claim/file-supplemental-claim-form-20-0995/contact-information) _should_ use this pattern, so you could [use this as a reference](https://github.com/department-of-veterans-affairs/va.gov-team/issues/52046).
    - **Also see:** Patterns > multiple responses > [multi-page](https://design.va.gov/patterns/ask-users-for/multiple-responses#how-to-design-and-build---multi-page)

cc: @coforma-terry @SarahKay8

**Comment by coforma-jamie** [comment]
Hello @Midge-dev! We're going to close this ticket since CAIA has provided the guidance you need to move forward. Use [this checklist](60647#issuecomment-1602771216) from @sara-amanda as you continue this work. Please feel free to reach out. Thank you!
